step2-01/README.md

```diff
@@ -122,51 +151,53 @@ async function someFunctionAsync() {
 }
 ```
 
-All functions that are marked `async` return a `Promise` automatically. This previous example returned a `Promise<string>`, and can be used like this:
+All functions that are marked `async` return a `Promise` automatically. The previous example returned a `Promise<string>`, and can be used like this:
 
 ```ts
 someFunctionAsync().then(result => {
   console.log(result);
 });
 ```
 
+> For more information, see [this article](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function).
+
 # Exercise
 
-Please complete all exercises inside the `exercise/src` folder unless otherwise specified in the exercises below. First, open up [Step2-01 exercise page](http://localhost:8080/step2-01/exercise/) to see the results while you're implementing things.
+Please complete all exercises inside the `exercise/src` folder unless otherwise specified. First, open up [Step2-01 exercise page](http://localhost:8080/step2-01/exercise/) to see the results while you're implementing things.
 
 ## Modules
 
-1. Open up file called `index.ts` in VS Code
+1. Open the file called `index.ts` in VS Code
 
 2. Create another module file called `fibonacci.ts`
 
-3. Inside the file from (step 2), write a function called `fib(n)` that takes in a number and returns a the n-th Fibonacci number - be sure the specify the type of n
+3. Inside the file from (step 2), write a function called `fib(n)` that takes in a number and returns the `n`-th Fibonacci number (be sure the specify the type of `n`).
 
-> HINT: function fib(n: number) { return n <= 1 ? n : fib(n - 1) + fib(n - 2); }
+> HINT: `function fib(n: number) { return n <= 1 ? n : fib(n - 1) + fib(n - 2); }`
```

Comment:
Do we really want to provide the whole function implementation? :)

Reply:
@micahgodbolt wanted it!

Reply:
Fair enough. It might feel too much like an interview warmup if we asked people to implement this off the top of their heads. :P

Reply:
YES!